readthedocs/templates/base.html

```diff
@@ -85,17 +85,13 @@
 
         {% block footer-content %}
         <p>
-        Copyright {% getCopyrightYears %}.
-
-        {% blocktrans with "<a href='http://ericholscher.com/'>Eric Holscher</a>" as eric and "<a href='http://charlesleifer.com/'>Charles Leifer<a>" as charles and "<a href='http://bobbygrace.info/'>Bobby Grace</a>" as bobby and "<a href='http://djangodash.com/'>Django Dash</a>" as djangodash %}
-          Created by {{ eric }}, {{ charles }}, and {{ bobby }} for the 2010 {{ djangodash }}.
-        {% endblocktrans %}
```

Comment:
Aye -- this should link to the team page in the docs, and/or the GH contributors page.